docs/antora.yml

```diff
@@ -11,6 +11,7 @@ asciidoc:
     compatible-centos-stream: 'CentOS Stream 9.x'
     compatible-rhel7: '7.x'
```

Comment:
Do we know whether we've decided to dump rhel7? Seems like we should. Might want to check with @jeffgdotorg to be sure.

Reply:
It would be great to remove that tab, as it messes up the layout. :-)